Builder.Andrew.PCTechnician/Builder.Andrew.PCTechnician/tests/TechnicianTest.cs

```diff
@@ -0,0 +1,48 @@
+using Builder.Andrew.PCTechnician.src;
+using Builder.Andrew.PCTechnician.src.Concrete_Builders;
+using System;
+using Xunit;
+
+namespace Builder.Andrew.PCTechnician
+{
+    public class TechnicianTest
+    {
+        private readonly Workshop _workshop = new Workshop();
+
+        [Fact]
+        public void TestDesktopBuilder()
```

Comment:
Let's work on how you name tests.  IMO, tests should read like sentences.  Test code is still code, and you should take the same care in naming and writing tests that you would for production code.

My personal naming strategy:  `WhenDoingX_UnderTheseConditions_ExpectThis` (e.g. `WhenSchedulingATask_WithUrgentTasks_ItSchedulesThemForImmediateExecution` or `WhenSchedulingATask_WithAnInvalidDate_ItThrowsAnArgumentException`).

```suggestion
        public void WhenBuildingADesktop_ItCreatesADesktopWithDesktopComponents()
```

In this case, you may not need to include the middle portion of the test name because there are no special conditions.